uiowa/uiowa
# Off-brand theme

# How to test

- `ddev blt ds --site=default`
- `ddev blt frontend`
- Test out changing settings to "Gray" on https://default.uiowa.ddev.site/admin/appearance/settings/uids_base. 
- Test backgrounds https://sandbox.prod.drupal.uiowa.edu/backgrounds-0. 

<img width="653" alt="Screenshot 2023-03-02 at 9 23 54 AM" src="https://user-images.githubusercontent.com/1036433/222471713-0d2ca5f0-4218-4971-a320-2dd66ce39334.png">


- Test out all the header options at all browser breakpoints.
  - Test with top-links (https://default.uiowa.ddev.site/admin/structure/menu/manage/top-links) You will need to add some links.
    - Test all variations with top links and then disable them and test all header variations.  The top bar should not display if there are no top links being used and the header type is set to "display below". 
  - Test site name above and site name below options with:
    - Horizontal nav
    - Toggle nav
      - Test sticky header option
    - Meg nav

Uncheck this and upload an image below.  And re-test all the options listed above. 

<img width="457" alt="Screenshot 2023-03-02 at 9 26 00 AM" src="https://user-images.githubusercontent.com/1036433/222472280-b4cf8070-1d44-41c3-8f36-2954b0c0e3a7.png">

### Logos for testing

<details>

![ill-01](https://user-images.githubusercontent.com/1036433/222470945-05bfeed1-e678-4826-a6e1-f26e867614d7.png)
![ileb_logo](https://user-images.githubusercontent.com/1036433/222470964-4c637b35-cdd4-4afe-8acb-333960bee820.png)

</details>

# Todo 
- [ ] Update this with UIDS approved prs. 
- [ ] Set variables on some of the webform opacity scss. 
- [x] Browser test this a lot. 
- [x] Some background patterns are still gold.
